src/ubx.cpp

```diff
@@ -2219,6 +2234,25 @@ GPSDriverUBX::payloadRxDone()
 		ret = 1;
 		break;
 
+	case UBX_MSG_RXM_RTCM:
+		UBX_TRACE_RXMSG("Rx RXM-RTCM");
+
+		if ((_buf.payload_rx_rxm_rtcm.flags & UBX_RX_RXM_RTCM_CRCFAILED_MASK) == 1) {
+			_gps_position->rtcm_crc_failed = true;
+			_rtcm_crc_failed_count++;
```

Comment:
Can you directly use _gps_position->rtcm_crc_failed_count?
And why is there both a counter and a flag?
Isn't 32 bits enough?